src/dscanner/analysis/del.d

```diff
@@ -15,30 +15,42 @@ import dsymbol.scope_;
  */
 extern(C++) class DeleteCheck(AST) : BaseAnalyzerDmd
 {
-	// alias visit = BaseAnalyzerDmd!AST.visit;
```

Comment:
Delete unused code, do not leave it commented.

Reply:
It is deleted. You commented on the old file

Reply:
Argh, nevermind.